internal/node-patches/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@bazel/node-patches",
+  "version": "1.0.0",
+  "description": "Runtime modifications to nodejs internals to help node/npm distributed programs run in bazel",
+  "main": "build/src/index.js",
+  "scripts": {
+    "test": "c8 mocha -r source-map-support/register build/test/fs/*.js build/test/subprocess/test.js",
+    "check": "prettier -c register.js && gts check",
```

Comment:
let's remove formatting and linting until we reconcile how that interacts with the monorepo-wide tooling